### PR TITLE
fix(generator-pie-component): DSW-000 skips visual tests

### PR DESCRIPTION
**Pull request by raoufswe**
## Describe your changes (can list changeset entries if preferable)
- as raised by [this request ](https://github.com/justeattakeaway/pie/pull/707#discussion_r1288096397), we should skip visual tests to avoid any unnecessary usage of snapshots 
- fixes a bug where the scope of commit messages is overridden by the ticket id

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [ ] If it is a component change, I have reviewed the Storybook preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: packages/components/pie-divider/test/visual/pie-divider.spec.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import percySnapshot from '@percy/playwright';
 import { PieDivider, DividerProps } from '@/index';
 
 test.describe('PieDivider - Visual tests`', () => {
-    test('should display the PieDivider component successfully', async ({ page, mount }) => {
+    test.skip('should display the PieDivider component successfully', async ({ page, mount }) => {
```
Comment (kevinrodrigues) [inline packages/components/pie-divider/test/visual/pie-divider.spec.ts:7]: I think you can move this one level up to the describe so the whole thing skips?

Comment (raoufswe) [reply packages/components/pie-divider/test/visual/pie-divider.spec.ts]: done

**Review by jamieomaguire — DISMISSED** [review]

**Review by kevinrodrigues — DISMISSED** [review]
🎖️

**Comment by fernandofranca** [comment]
Awesome work!
Could you add to the PR description why these changes are necessary?

**Review by fernandofranca — DISMISSED** [review]

**Review by fernandofranca — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Pull request merged**